pkg/controller/controller_test.go

```diff
@@ -527,7 +526,7 @@ func fakeClaim(name, namespace, claimUID, capacity, boundToVolume string, phase
 			AccessModes: []v1.PersistentVolumeAccessMode{v1.ReadWriteOnce, v1.ReadOnlyMany},
 			Resources: v1.ResourceRequirements{
 				Requests: v1.ResourceList{
-					v1.ResourceName(v1.ResourceStorage): resource.MustParse(capacity),
+					v1.ResourceName(v1.ResourceStorage): resource.MustParse(strconv.FormatInt(capacity, 10)),
```

Comment:
Use `NewQuantity()` instead of converting int to string and parsing it.

Reply:
Addressed here and also in the `generatePVCForProvisionFromPVC` function.

When addressing this I did find that the addition of t.Parallel required a local assignment of tc, as otherwise every parallel test was executing the same tc (the last one as per the random order of tests). Would like a review of the change specific to this a bit more carefully, https://github.com/kubernetes-csi/external-provisioner/pull/411/files#diff-a846267007f026d95b52f6baaf249948R2974-R2978